### PR TITLE
Clarifying how shared colorbars are created

### LINKED ISSUE #276: Matplotlib content on setting consistent colorbars
Producing two (or more) contour or pcolor plots of different arrays but with the same colorscale (for fair comparison) is something that comes up very frequently but isn't always obvious how to do with matplotlib.

I think our tutorials should have a simple example for how to achieve this, probably in the first intro notebook.

We have examples of more advanced colorbar control, but I think there's a gap on this important day-to-day plotting need.

### PR DESCRIPTION
**Pull request by hCraker**
This PR closes #276 

During the recent Pythia Hacakthon, I made a [cookbook](https://github.com/ProjectPythiaCookbooks/mpl-colorbar-cookbook) explaining how to make shared colorbars in matplotlib and how to choose accessible colormaps. It was determined in the linked issue that the information in the cookbook is better suited for pythia-foundations. This PR adds information that was in the cookbook to the relevant existing sections of pythia-foundations.

### CODE REVIEW

File: core/matplotlib/annotations-colorbars-layouts.ipynb [hunk 1]
```diff
@@ -426,11 +426,54 @@
    "id": "1662bf3c",
```
Comment (r-ford) [inline core/matplotlib/annotations-colorbars-layouts.ipynb:426]: <blockquote>Thus, it does matter in this particular example which output from <code>contourf</code> is used to make the colorbar.</blockquote><p>I think an emphasis on <em>does </em>would help, since I read it as "doesn't" at first.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (r-ford) [inline core/matplotlib/annotations-colorbars-layouts.ipynb:426]: <blockquote>In fact, we used this when ploting the histograms just above!</blockquote><p>Is this section meant to go after the two plots below? I think it would flow better that way.</p><p> </p><p>Also "ploting" --&gt; "plotting"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Review by r-ford — APPROVED** [review]
Looks good!

File: core/matplotlib/annotations-colorbars-layouts.ipynb (hunk 1 above)
Comment (mgrover1) [inline core/matplotlib/annotations-colorbars-layouts.ipynb:426]: <p>Agree with previous comment - maybe reword to "This functionality was utilized with the previous histogram exercise"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small suggestion - other than that, it looks great! Thanks @hCraker

**Review by mgrover1 — APPROVED** [review]
Great! Thanks @h
